src/basics.php

```diff
@@ -120,6 +122,20 @@ function breakpoint(): ?string
     }
 }
 
+if (!function_exists('table') && (PHP_SAPI === 'cli' || PHP_SAPI === 'phpdbg')) {
+    /**
+     * REPL convenience function to load a table
```

Comment:
Would be nice to add explicit disclaimer too to not use this function in app/plugin code.

Reply:
How about:

``Do **NOT** use this function in your app or plugin code?``

Reply:
I would still remove this here into its own separate file, which can have this as file doc block or warning on the top.

Reply:
Moving it in separate file would allow us to load the file using `require-dev` and address our concerns of it's misuse.

Reply:
`breakpoint()` could also be moved to the new file then.

Reply:
Yep, that was my thought, too.

I also had in mind to contribute the pretty print function @lorenzo used in one of his workshops once.
But I couldn't find his code, unfortunately.
Such a function could be placed there, too.

Reply:
@ADmad added the disclaimer.

@dereuromark  & @ADmad moved both functions into ``repl_functions.php``.

> Moving it in separate file would allow us to load the file using require-dev and address our concerns of it's misuse.

This can be done in cakephp/app, once this PR is ready/merged.